units/event_metajob.go

```diff
@@ -239,6 +239,17 @@ func (j *eventMetaJob) dispatch(ctx context.Context, notifications []notificatio
 	return catcher.Resolve()
 }
 
+// dispatchUnprocessedNotifications gets unprocessed notifications
+// leftover by previous runs and dispatches them
+func (j *eventMetaJob) dispatchUnprocessedNotifications(ctx context.Context) error {
+	unprocessedNotifications, err := notification.FindUnprocessed()
+	if err != nil {
+		return errors.Wrap(err, "can't find unprocessed notifications")
```

Comment:
should we error if not found?

Reply:
if there are no unprocessed notifications `dispatch` will noop.